Dockerfile

```
@@ -0,0 +1,37 @@
+# Installation stage
+# Offical Node.js Alpine base image
+FROM node:lts-alpine AS base
+
+# Set working directory
+WORKDIR /base
+
+# Copy package.json and package-lock.json for utilising Docker cache 
+# to save re-installing dependencies if unchanged
+COPY package*.json yarn.lock ./ 
+
+# Install dependencies only needed
+RUN yarn install --production --frozen-lockfile
+
+# Copy installed dependencies to current working directory
+COPY . .
+
+# Build stage
+FROM base AS builder
+ENV NODE_ENV=production
+WORKDIR /build
+COPY --from=base /base ./
```

A slash isn't needed since we specify only one location 😅
```suggestion
COPY --from=base /base .
```

---

I intend to copy to `./` like I copied in `installation stage`

---

Actually, I intend to copy it to `./` like in `installation stage`